### PR TITLE
Small design fixes for "Cancel login" link on the 2FA challenge page

### PR DESCRIPTION
**Pull request by ChristophWurst**
Small styling enhancement by @jancborchardt 

Before
bildschirmfoto von 2016-08-27 10-53-16

After
bildschirmfoto von 2016-08-27 10-53-27

### CODE REVIEW

**Comment by LukasReschke** [comment]
:+1:

**Comment by mar1u5** [comment]
👍

File: core/css/styles.css [hunk 1]
```diff
@@ -38,8 +38,10 @@ body {
 	display: inline-block;
 }
 
-a.two-factor-cancel {
-	color: #fff;
+.two-factor-cancel {
+	display: inline-block;
+	padding: 12px;
+	color: rgba(255, 255, 255, .75);;
```
Comment (nickvergessen) [inline core/css/styles.css]: double `;`

**Comment by jancborchardt** [comment]
@nickvergessen you know you can also fix that directly in the web interface ;) just fixed it. :)

:+1:

**Pull request merged**